### DISCUSSION TITLE
[Question]: does greeting support multiline message?

### DISCUSSION THREAD

**Discussion by elvis-cai** [danny-avila/LibreChat#2877] [Q&A]
### What is your question?

we want to add multiline announcement as greeting msg to users, we tested a few examples, unfortunately, multi-line is not interpret correctly

### More Details

this is the config we are referring to https://www.librechat.ai/docs/configuration/librechat_yaml/object_structure/model_specs#greeting

### What is the main subject of your question?

UI

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/38489894/d5dfda6e-5c6d-462c-a540-94887b0c3f1c)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
No it does not. It’s meant to be a short greeting, you are better off editing the code for something like this.

**Reply by elvis-cai** [reply]
thanks @danny-avila , it works after editing the `Landing.tsx` file.
